src/Microsoft.Diagnostics.Monitoring.WebApi/Controllers/DiagController.Metrics.cs

```diff
@@ -25,7 +25,6 @@ partial class DiagController
         [ProducesWithProblemDetails(ContentTypes.ApplicationJsonSequence)]
         [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status429TooManyRequests)]
         [ProducesResponseType(typeof(void), StatusCodes.Status202Accepted)]
-        [RequestLimit(LimitKey = Utilities.ArtifactType_Metrics)]
```

Comment:
I know that we want to route everything through the operations service eventually, but removing these without doing that leaves all of the routes in a state of allowing unlimited requests. I would ask that you add these attributes and the middleware back until the route is actually registered in the operation service.